config/rbac/role.yaml

```diff
@@ -6,6 +6,26 @@ metadata:
   creationTimestamp: null
   name: manager-role
```

Comment:
Have you tried to deploy the operator via a Pod?

It seems the role lacks permissions to manage Pods and Services. If you have been running the operator locally (as a process in your machine) it makes use of the permissions of the K8s user you are logged in with, usually an admin-level one, so you usually do not reproduce possible existing permission level issues in that case.

Reply:
I've been running the operator locally with `make run`. I'll try to run it in kube and fix the permissions.

Reply:
There's a problem with building the docker image. I'll address this in a separate PR.

Reply:
ok :+1: